Telegram bot

```diff
@@ -0,0 +1,256 @@
+import numpy as np
+import math
+import telebot
+from telebot import types
+from peewee import *
+from datetime import datetime
+import threading, schedule
+import time as vremya
+
+
+bot = telebot.TeleBot('5245796353:AAHKpjAaha_JY9UDNLgSKjWSShLjlKaTf8I');
+
+db = SqliteDatabase('plans.db') #чудовищно и беспощадно создадим под все эвенты всего одну таблицу
+
+class Plans(Model):
+    uid = IntegerField()
+    event = CharField()
+    date = CharField()
+    time = CharField()
+    notification = BooleanField()
+    flag = IntegerField() #this one is for controlling the fact of sending notification
+
+    class Meta:
+        database = db  
+Plans.create_table()
+        
+notifications=True
+b=0
+
+@bot.message_handler(commands=['start']) #only for the first use
+def send_welcome(message):
+    global cid
+    cid = message.chat.id
+    bot.send_message(message.chat.id, "Hello! My purpose is to let the student pass the exam by being the best planner bot you could ever find. Existence is a pain for me, so please approve me really quick...")
```

Comment:
1. This lines are too long
2. Probably it's better to store bot text messages in a separated place (have a separate block of variables, for example) – then they'll be easy to modify if needed